### PR TITLE
Replace node-fetch package with node-fetch-with-proxy to support customers using proxy

### LINKED ISSUE #1193: Update orchestrator-core reference to 4.13.0 version

### PR DESCRIPTION
**Pull request by tsuwandy**
Fixes #1193

### CODE REVIEW

**Review by daveta — APPROVED** [review]

**Review by taicchoumsft — APPROVED** [review]

File: packages/chatdown/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "latest-version": "^4.0.0",
     "mime-types": "^2.1.18",
     "minimist": "^1.2.0",
-    "node-fetch": "~2.6.0",
+    "node-fetch-with-proxy": "^0.1.4",
```
Comment (munozemilio) [inline packages/chatdown/package.json:21]: Does this support https over http? If this is to support composer please sync with @cosmicshuai to have the same behavior as the lu lib

**Review by munozemilio — CHANGES_REQUESTED** [review]
Small question on npm package behavior.

File: packages/chatdown/package.json (hunk 1 above)
Comment (tsuwandy) [reply packages/chatdown/package.json:21]: Yes, it's for composer and support https/http.  I updated all packages in bf cli.

**Comment by tsuwandy** [comment]
@feich-ms, @munozemilio, any idea why the lg package errors out in the validation check?

**Comment by boydc2014** [comment]
@cosmicshuai can you help check this one?

**Comment by tsuwandy** [comment]
@cosmicshuai, these are the errors:

FAILURE (1)
================================
@microsoft/bf-lg-cli (6 minutes 48.8 seconds)
D:\src\botframework-cli\packages\lg\test\fixtures\testcase\3.lg:[Error] D:\src\botframework-cli\packages\lg\test\fixtures\testcase\3.lg line 5:4 - line 5:7: Invalid structure body: 'abc'. Body can include <PropertyName> = <Value> pairs or ${reference()} template reference.
D:\src\botframework-cli\packages\lg\test\fixtures\testcase\3.lg:[Error] D:\src\botframework-cli\packages\lg\test\fixtures\testcase\3.lg line 10:0 - line 11:1: Invalid structure body. Body cannot be empty.
D:\src\botframework-cli\packages\lg\test\fixtures\testcase\3.lg:[Error] D:\src\botframework-cli\packages\lg\test\fixtures\testcase\3.lg line 15:2 - line 15:17: Error occurred when parsing expression 'NOTemplate()'. NOTemplate does not have an evaluator, it's not a built-in function or a custom function.
D:\src\botframework-cli\packages\lg\test\fixtures\testcase\3.lg:[Error] D:\src\botframework-cli\packages\lg\test\fixtures\testcase\3.lg line 20:9 - line 20:24: Property  'Text=${NOTemplate()}': Error occurred when parsing expression 'NOTemplate()'. NOTemplate does not have an evaluator, it's not a built-in function or a custom function.
D:\src\botframework-cli\packages\lg\test\fixtures\testcase\3.lg:[Error] D:\src\botframework-cli\packages\lg\test\fixtures\testcase\3.lg line 24:0 - line 24:10: Invalid structure name: 'Activity%'. name should start with letter/number/_ and can only contains letter/number/./_.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @microsoft/bf-lg-cli@1.0.0 posttest: `eslint . --ext .ts --config .eslintrc`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @microsoft/bf-lg-cli@1.0.0 posttest script.

**Comment by cosmicshuai** [comment]
@tsuwandy It is a lint issue. I committed a change to fix it.

**Comment by cosmicshuai** [comment]
image
I did a simple test to validate node-fetch-proxy works for the cli. Enabled a firewall rule to block all connections to port 443. Then set https_proxy to my proxy server. After proxy was set, I run the command to download the base model. The model files were successfully downloaded.

**Comment by boydc2014** [comment]
@tsuwandy looks like there is still one last error blocking CI, is this something you can help check in your day time?

**Comment by daveta** [comment]
This is referencing package with security problems.

**Pull request closed without merging**